openlattice.yaml

```diff
@@ -1955,8 +1955,7 @@ paths:
             application/json:
               schema:
                 $ref: "components/dataSearchResult"
-
-  /datastore/search/{entitySetId}/{entityKeyId}:
+  /datastore/search/{entitySetId}/{entityId}:
```

Comment:
Are you renaming `entityKeyId` to `entityId` everywhere? If so, what are you calling `entityId`s?

Reply:
Hm I guess not since I see more `entityKeyId`s below -- so why was this renamed? (and the one below)

Reply:
Because it's entityId in SearchApi.java: https://github.com/openlattice/api/blob/develop/src/main/java/com/openlattice/search/SearchApi.java#L69

Reply:
oh whoops -- that should be `entityKeyId` in SearchApi

Reply:
PRs to fix this:
- https://github.com/openlattice/api/pull/389
- https://github.com/openlattice/datastore/pull/460

Reply:
aaaand fixed! so this should stay as `entityKeyId`

Reply:
ok fixed !
How about this one:
https://github.com/openlattice/api/blob/51b70298be76a7758f56ee47cb556f2f43fe43af/src/main/java/com/openlattice/data/EntityKey.java#L43

Reply:
that one is an actual entityId :) 

An easy way to distinguish is that entityIds are Strings, and entityKeyIds are UUIDs -- you can look at the actual variables that they map to in order to check